otomoto/spiders/otomoto.py

```diff
@@ -37,30 +37,35 @@ def parse_car_page(self, response):
             'Marka pojazdu': 'brand',
             'Model pojazdu': 'model',
             'Rok produkcji': 'year',
+            'Wersja': 'version',
             'Przebieg': 'mileage',
             'Pojemność skokowa': 'capacity',
             'Moc': 'horse_power',
             'Rodzaj paliwa': 'fuel_type',
             'Skrzynia biegów': 'transmission',
             'Typ': 'type',
             'Liczba drzwi': 'number_of_doors',
+            'Kraj pochodzenia': 'origin_country',
             'Kolor': 'color',
             'Pierwszy właściciel': 'first_owner',
             'Bezwypadkowy': 'no_accidents',
             'Serwisowany w ASO': 'aso',
             'Stan': 'condition',
         }
         loader = OtomotoCarLoader(OtomotoItem(), response=response)
-        for params in response.css('.offer-params__item'):
 
+        for params in response.css('.offer-params__item'):
             property_name = params.css('.offer-params__label::text').extract_first().strip()
+            # print(property_name)
             if property_name in property_list_map:
-                css = params.css('div::text').extract_first().strip()
+                css = params.css('.offer-params__value::text').extract_first().strip()
```

Comment:
My selector works as well.

Reply:
You're right, I have restored your version. Mine seemed more readable to me.

Reply:
What params? For me it works well.  Can you show two jsons?

Reply:
I merged that, I think it's better to use more specific selector. Thanks